### ISSUE TITLE
Seems dead

### ISSUE THREAD

**Issue by pyrofolium** [sdiehl/write-you-a-haskell#108]
You ever going to finish this? I've been waiting for this for years.

**Comment by tbenst** [comment]
Just wanted to chime in that sdiehl has no obligations to us to finish this, and we are not entitled to it.

That said, I too would eagerly anticipate it being finished and think this book is an amazing contribution not just to open source or Haskell, but computer science at large. Thank you sdiehl—if we can support you in any way, financially or otherwise, to finish this, let us know :)

**Comment by FranklinChen** [comment]
Also see #93

**Comment by pyrofolium** [comment]
>Just wanted to chime in that sdiehl has no obligations to us to finish this, and we are not entitled to it.

That's right and you make sure you remember that. sdiehl is under no obligation finish it. Please don't act as if you're entitled to it.
